geolocalize_nominatim/models/res_partner.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2004-2015 Odoo S.A. (original module : base_geolocalize)
+# © 2018 Le Filament (<http://www.le-filament.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+import json
+import urllib2
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError
+
+
+def geo_find(addr):
+    if not addr:
+        return None
+    url = 'https://nominatim.openstreetmap.org/search.php/'
```

Comment:
No need for three assignments. Do one, either using line continuation, or by putting the three partial strings between `(` and `)`.